rebar.config

```diff
@@ -1,2 +1,17 @@
-{erl_opts, [debug_info]}.
-{deps, []}.
\ No newline at end of file
+{erl_opts, [debug_info,
+            {parse_transform, lager_transform}]}.
```

Comment:
We need this because there are calls to `lager:error` or `lager:info` functions in the source code. These functions are available only after running the parse transform. Without the parse transform xref was complaining.

Switching to logger is a good idea!